templates/front/my_checks.html

```diff
@@ -166,6 +166,18 @@ <h4 class="update-timeout-title">Name and Tags</h4>
                     </div>
                     <div id="nag-slider"></div>
 
+                    <div class="checkbox update-timeout-info text-left">
+                        <label>
+                            <input
+                                name="nag_mode"
+                                type="checkbox"
+                                data-toggle="tooltip"
+                                title="When the nag mode is selected, the user will be nagged until the check resolved"
+                                {% if nag_mode %} checked {% endif %}>
```

Comment:
Here is where the problem arises; we need to keep the checkbox pre-selected if the user turned on Nag mode.

Reply:
Found the solution as `checks.nag_mode`